desk/app/channels.hoon

```diff
@@ -11,10 +11,15 @@
 /-  c=channels, g=groups, ha=hark, activity
 /-  meta
 /+  default-agent, verb, dbug, sparse, neg=negotiate, imp=import-aid
-/+  utils=channel-utils, volume, s=subscriber
+/+  utils=channel-utils, volume, s=subscriber, em=emojimart
+::
 ::  performance, keep warm
 /+  channel-json
 ::
+::  compile all marks
+::XX this is not actually compiling??
+:: /~  marks  *  /mar/channel
+::
 =/  verbose  |
 %-  %-  agent:neg
     :+  notify=&
```

Comment:
So, under this, `[%channels-server^[~.channels^%2 ~ ~] ~ ~]`.

(And arguably the self-protocol should just go, we don't use that as intended here.)